### ISSUE TITLE
`.at-*-full-width` with `.equalize` can produce a odd display

### ISSUE THREAD

**Issue by jeremyBass** [washingtonstateuniversity/WSU-spine#226]
I have run into a condition we have not set forth a stance about.  When you do something like 

```html
	<div class="row halves equalize at-large-full-width">			
		<div class="column one">
			<div style="height:100px;"></div>
		</div>
		<div class="column one">
			<div style="height:500px;"></div>
		</div>
	</div>
```
What will happen is you have a design that is meant for side by side, but when the column breaks the `.equalize` function doesn't do anything or react.  There is no out of the box way, that I could find, that addresses this condition where the design is now visual awkward due to the big gap that is formed from equalizing the two rows.  

I believe we can deal with this in a backward compatible way by adding the ability to set `at-<size>-equalize` .  We can then alter the function `equalize()` to do an `.is()` to check if it's apart of a `at-` namespace and then add in some sizing logic based of the body class.  Should only add about 4-5 lines of light weight js, no css, and about 10 new virtual namespaces.

The goal would to just alter starting at [ui.spine.framework.js#L187](https://github.com/washingtonstateuniversity/WSU-spine/blob/develop/scripts/ui.spine.framework.js#L187) and that would be the only area change.  Since this would be 100% backwards compatible change we should be good on that front.

**Comment by jeremyBass** [comment]
Also turns out that we can end up on resize in a reset set when in a mobile view.

Mobile view example, 
![2015-01-06_0956](https://cloud.githubusercontent.com/assets/43837/5633255/55a1b494-958a-11e4-8d7b-114826267346.png)
After resize to desktop
![2015-01-06_0956_001](https://cloud.githubusercontent.com/assets/43837/5633272/6ffd73b4-958a-11e4-8312-c61eed05ecb1.png)
So what happens is that the media queries at a point with null out the affects of `.equalize` with `if($(window).width() <= 792) {` from the function, but that seems to be very arbitrary and cuffs the designer.  here is an example, 

When in desktop view, which this is the same size at a mobile view and would want it to keep being equalized.  
![2015-01-06_1002](https://cloud.githubusercontent.com/assets/43837/5633372/5cd82d96-958b-11e4-8e8c-82056c122c9d.png)

But when I get to the point were we check for the `792px` width of the viewport I end up with this even thought the size wasn't smaller, and I would still wish it to match
![2015-01-06_1003](https://cloud.githubusercontent.com/assets/43837/5633388/7fc4e0ec-958b-11e4-9946-acdd09123948.png)


Just a few more notes on this item.

**Comment by jeremyfelt** [comment]
Is this fixed via #262?

**Comment by natejacobson** [comment]
It is fixed, @jeremyfelt. @jeremyBass is still planning on extending my minimalistic solution, but it should be good for the time being.
